acceptance/tests/resource/service/puppet_mcollective_service_management.rb

```diff
@@ -0,0 +1,44 @@
+test_name "Puppet and Mcollective services should be manageable with Puppet"
+
+confine :except, :platform => 'windows'
+confine :except, :platform => 'solaris' # See PUP-5262
```

Comment:
we all should.  i've been meaning to socialize it more.   it's amazing.  Chris Cowell did it one day out of the blue and PRd it into beaker.